src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+:root {
+  margin: 0;
+}
+
+.stars {
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes`. If you need to reset default margins and paddings, it's better to do it specifically for the `body` or `html` tags, not using the `:root` selector.